site/src/templates/componentPage.tsx

```diff
@@ -42,6 +43,14 @@ export default ({ data, pageContext, location }) => {
     allPages.filter(el => el.node.frontmatter.navTitle !== "Overview")
   )
 
+  const renderStorybookIFrame = () => {
+    return md.frontmatter.demoStoryId ? (
```

Comment:
Do you think it would be worthwhile logging an error to the console here? Just thinking about what might be useful for a non-developer trying to update the docs and not seeing their example. 🤔

Reply:
Superb idea. Will do 👍